src/main/resources/assets/mooblooms/lang/de_de.json

```diff
@@ -0,0 +1,168 @@
+	"text.autoconfig.mooblooms.option.poppyMoobloom.minGroupSize": "Minimale Gruppengröße",
+	"text.autoconfig.mooblooms.option.poppyMoobloom.maxGroupSize": "Maximale Gruppengröße",
+	"text.autoconfig.mooblooms.option.poppyMoobloom.spawnBlocks": "Spawn Blöcke",
+	"text.autoconfig.mooblooms.option.blueOrchidMoobloom": "Blaue Orchideenblumenkuh",
+	"text.autoconfig.mooblooms.option.blueOrchidMoobloom.spawnEnabled": "Kann spawnen",
+	"text.autoconfig.mooblooms.option.blueOrchidMoobloom.weight": "Gewicht",
+	"text.autoconfig.mooblooms.option.blueOrchidMoobloom.minGroupSize": "Minimale Gruppengröße",
+	"text.autoconfig.mooblooms.option.blueOrchidMoobloom.maxGroupSize": "Maximale Gruppengröße",
+	"text.autoconfig.mooblooms.option.blueOrchidMoobloom.spawnBlocks": "Spawn Blöcke",
+	"text.autoconfig.mooblooms.option.alliumMoobloom": "Zierlauchblumenkuh",
+	"text.autoconfig.mooblooms.option.alliumMoobloom.spawnEnabled": "Kann spawnen",
+	"text.autoconfig.mooblooms.option.alliumMoobloom.weight": "Gewicht",
+	"text.autoconfig.mooblooms.option.alliumMoobloom.minGroupSize": "Minimale Gruppengröße",
+	"text.autoconfig.mooblooms.option.alliumMoobloom.maxGroupSize": "Maximale Gruppengröße",
+	"text.autoconfig.mooblooms.option.alliumMoobloom.spawnBlocks": "Spawn Blöcke",
+	"text.autoconfig.mooblooms.option.oxeyeDaisyMoobloom": "Margeritenblumenkuh",
+	"text.autoconfig.mooblooms.option.oxeyeDaisyMoobloom.spawnEnabled": "Kann spawnen",
+	"text.autoconfig.mooblooms.option.oxeyeDaisyMoobloom.weight": "Gewicht",
+	"text.autoconfig.mooblooms.option.oxeyeDaisyMoobloom.minGroupSize": "Minimale Gruppengröße",
+	"text.autoconfig.mooblooms.option.oxeyeDaisyMoobloom.maxGroupSize": "Maximale Gruppengröße",
+	"text.autoconfig.mooblooms.option.oxeyeDaisyMoobloom.spawnBlocks": "Spawn Blöcke",
+	"text.autoconfig.mooblooms.option.cornflowerMoobloom": "Kornblumenblumenkuh",
+	"text.autoconfig.mooblooms.option.cornflowerMoobloom.spawnEnabled": "Kann spawnen",
+	"text.autoconfig.mooblooms.option.cornflowerMoobloom.weight": "Gewicht",
+	"text.autoconfig.mooblooms.option.cornflowerMoobloom.minGroupSize": "Minimale Gruppengröße",
+	"text.autoconfig.mooblooms.option.cornflowerMoobloom.maxGroupSize": "Maximale Gruppengröße",
+	"text.autoconfig.mooblooms.option.cornflowerMoobloom.spawnBlocks": "Spawn Blöcke",
+	"text.autoconfig.mooblooms.option.witherRoseMoobloom": "Wither Rosenlumenkuh",
+	"text.autoconfig.mooblooms.option.witherRoseMoobloom.spawnEnabled": "Kann spawnen",
+	"text.autoconfig.mooblooms.option.witherRoseMoobloom.weight": "Gewicht",
+	"text.autoconfig.mooblooms.option.witherRoseMoobloom.minGroupSize": "Minimale Gruppengröße",
+	"text.autoconfig.mooblooms.option.witherRoseMoobloom.maxGroupSize": "Maximale Gruppengröße",
+	"text.autoconfig.mooblooms.option.witherRoseMoobloom.spawnBlocks": "Spawn Blöcke",
+	"text.autoconfig.mooblooms.option.witherRoseMoobloom.damagePlayers": "Kann Spielern schaden",
+	"text.autoconfig.mooblooms.option.suncower": "Sonnenblumenkuh",
```

Comment:
Sonnenkuh or change "entity.mooblooms.suncower" and "item.mooblooms.suncower_spawn_egg"